requirements.txt

```diff
@@ -0,0 +1,9 @@
+requests
```

Comment:
This file becomes redundant because the packages are mentioned in `setup.py`. If you are concerned about the development setup, populate this file only with `-e .[dev]`, and mention `dev` dependencies if there are any in `setup.py`